tensorflow/tools/docker/parameterized_docker_build.sh

```diff
@@ -299,6 +317,26 @@ else
   sed "s/^RUN git clone --branch=.* --depth=1/RUN git clone --branch=${TF_DOCKER_BUILD_DEVEL_BRANCH} --depth=1/" \
       "${ORIG_DOCKERFILE}" > "${DOCKERFILE}"
 
+  # Modify the devel Dockerfile to specify the MKL bazel build parameters
+  if [[ "${TF_DOCKER_BUILD_TYPE}" == "mkl" ]]; then
+    if sed -i -e 's/bazel build -c opt /bazel build -c opt --config=mkl /g' "${DOCKERFILE}"
```

Comment:
My comment still stands from before. I'd prefer if we could set the necessary Bazel options in a less hacky way.

Reply:
I think using sed is an optimal solution for setting multiple Bazel options in Docker file, it makes code neat and elegant. This file also uses sed in other place. Any better suggestion?

Reply:
@case540 The second option is using bazelrc file to pass Bazel options to Docker file, this method is used in
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/ci_build/Dockerfile.cpu.mpi